client/src/components/Public/Map/ClusterMarker/ClusterMarker.js

```diff
@@ -31,21 +32,25 @@ const ClusterMarker = ({
   zoomToLocation,
   onClick,
   openedAddressId,
+  MarkerLabel,
 }: ClusterMarkerProps) => {
-  const MarkerText = <span className="marker__text">{cluster.numPoints}</span>
   let className, children
   const selected = cluster.numPoints === 1 && cluster.points[0].address_id === openedAddressId
   if (zoom < ENTITY_ZOOM) {
-    className = cluster.numPoints === 1 ? 'simple-marker' : 'cluster-marker'
-    children = cluster.numPoints !== 1 && <span className="marker__text">{cluster.numPoints}</span>
+    className = (MarkerLabel === true)
```

Comment:
Related, this is the most confusing condition ever.

Reply:
this is something what i already inherited, i guess its ready for a makeover...

Reply:
well ok the reason now i remember was that if i wanted to switch to map-labels then i would need to edit only one string...

Reply:
changed to be more readable